### PR TITLE
[Inflector] Fix plural of "podium"

### LINKED ISSUE #44949: Pluralization of "Podium" is incorrect
### Symfony version(s) affected

All

### Description

When using the method `pluralize` of the class `Symfony\Component\String\Inflector\EnglishInflector` on the word "Podium" you end up with "Podia" when you should get "Podiums"

### How to reproduce

Execute pluralize('podium') you'll get 'Podia'

### Possible Solution

_No response_

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by W0rma**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| Tickets       | Fix #44949 
| License       | MIT
| Doc PR        |

### CODE REVIEW

**Comment by W0rma** [comment]
fabbot errors seem to be unrelated

File: src/Symfony/Component/Inflector/Inflector.php [hunk 1]
```diff
@@ -30,6 +30,9 @@ final class Inflector
         // Fourth entry: Whether the suffix may succeed a consonant
         // Fifth entry: singular suffix, normal
 
+        // podiums (podium)
```
Comment (ro0NL) [inline src/Symfony/Component/Inflector/Inflector.php:33]: IIUC it's generally prefered over podia? https://dictionary.cambridge.org/dictionary/english/podium

File: src/Symfony/Component/Inflector/Tests/InflectorTest.php [hunk 2]
```diff
@@ -259,6 +260,7 @@ public function pluralizeProvider()
             ['photo', 'photos'],
             ['piano', 'pianos'],
             ['plateau', ['plateaus', 'plateaux']],
+            ['podium', 'podiums'],
```
Comment (ro0NL) [inline src/Symfony/Component/Inflector/Tests/InflectorTest.php:264]: `['podium', ['podiums', 'podia']]`?

**Comment by mehdibo** [comment]
As @kaznovac mentioned:

> actually both podiums and podia are correct plurals of podium
see: https://www.merriam-webster.com/dictionary/podium

**Comment by fabpot** [comment]
@W0rma Do you want to finish this PR?

**Pull request closed without merging**